src/test/scala/ironoxide/FullIntegrationTest.scala

```diff
@@ -358,16 +358,15 @@ class FullIntegrationTest extends AsyncWordSpec with Matchers with EitherValues
       val data = ByteVector(List(1, 2, 3).map(_.toByte))
 
       val name = GroupName("a name")
-      val validGroupUUID = java.util.UUID.randomUUID.toString
+      val validGroupUUID = ju.UUID.randomUUID.toString
       val id = GroupId(validGroupUUID)
 
       // create a valid group then immediately encrypt to it
       val result = sdk
         .groupCreate(GroupCreateOpts(id, name))
-        .flatMap(
-          groupResult =>
-            sdk.advanced
-              .documentEncryptUnmanaged(data, DocumentEncryptOpts.withExplicitGrants(false, Nil, List(groupResult.id)))
+        .flatMap(groupResult =>
+          sdk.advanced
+            .documentEncryptUnmanaged(data, DocumentEncryptOpts.withExplicitGrants(false, Nil, List(groupResult.id)))
         )
```

Comment:
We need more tests here, it seems.

Reply:
I can write up some tests for these functions tomorrow, but we typically don't do very deep testing in this repo so I'll try to keep it light

Reply:
Added tests for all new functions